experimental/packages/otlp-transformer/package.json

```diff
@@ -49,7 +49,7 @@
   },
   "devDependencies": {
     "@opentelemetry/api": ">=1.0.0 <1.2.0",
-    "@opentelemetry/api-metrics": "~0.27.0",
+    "@opentelemetry/api-metrics": "0.28.0",
```

Comment:
peer dep above needs to be updated - or wait on #2904 and have this and a few other dependency nits fixed.

Reply:
merged 2904 thanks